flowzone.yml

```diff
@@ -686,6 +686,24 @@
     run: |
       aws eks update-kubeconfig --name "$(echo "${KUBE_CTX}" | awk -F'/' '{print $2}')"
 
+  - &publishSBOMArtifacts
+    name: Publish SBOM artifacts
+    uses: actions/upload-artifact@65462800fd760344b1a7b4382951275a0abb4808 # v4.3.3
+
+  - &publishSBOMToDependencyTrack
+    name: Publish SBOM To Dependency Track
+    if: ${{ vars.DTRACK_API != '' }}
```

Comment:
use  `${{ env.SERVER_HOASTNAME }}` everywhere or drop it and just use `${{ vars.DTRACK_API || 'default.hostname.org' }}`

Reply:
good catch